pennylane/circuit_drawer/utils.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2018-2021 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This module contains some useful utility functions for circuit drawing.
```

Comment:
Could it be worth adding these two functions to `pennylane/utils.py`? They are being used for circuit drawing here, but seem to be more general than this use case. It might be that they could be used for other purposes too in the future and having them here might "hide them".

Reply:
I'm concerned `pennylane/utils.py` is a hidden mishmash of stuff anyway.

 If we see a place elsewhere in pennylane that we would also like to use these functions, I'm all for moving them to `pennylane/utils.py`, but I don't want them getting lost. Placing them here ties them more closely to their use cases.